libafl/src/feedbacks/mod.rs

```diff
@@ -211,7 +226,14 @@ where
         self.second.init_state(state)?;
         Ok(())
     }
-
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn last_result(&self) -> Option<bool> {
+        return FL::last_result(&self.first, &self.second);
+    }
+    #[cfg(feature = "track_hit_feedbacks")]
+    fn get_hit_feedbacks(&self, list: &mut Vec<String>) {
```

Comment:
we don't usually have `get_` for names

Reply:
But let's change it to `append_hit_feedbacks` it captures better what's going on I think